src/components/utils/offers/FieldOptions.js

```diff
@@ -1,13 +1,30 @@
 /* istanbul ignore file */
 
 export default Object.freeze({
+    "ARTIFICIAL INTELLIGENCE": "Artificial Intelligence",
     "BACKEND": "Back-End",
-    "FRONTEND": "Front-End",
-    "DEVOPS": "Dev-Ops",
     "BLOCKCHAIN": "Blockchain",
-    "MACHINE LEARNING": "Machine Learning",
```

Comment:
Why did you remove Machine Learning?

Reply:
I changed into Artificial Intelligence, which is more broad (in backend as well)